src/App.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
-import "./App.css";
+// import "./App.css";
+// import SearchButton from "./SearchButton";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+// import SearchResults from "./SearchResults.JS";
+
+const data = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      {/* <header className="App-header">CYF Hotel</header> */}
+      <Heading />
       <Bookings />
+      <TouristInfoCards />
+      <Footer data={data} />
```

Comment:
this is good, you've imported your components and you're passing your address data as props to your `Footer` component